docs/data-sources/fabric_node_member.md

```diff
@@ -11,6 +11,15 @@ description: |-
 
 Data source for ACI Fabric Node Member
 
+## API Information ##
```

Comment:
Will this resource and datasource be manual or be migrated in time? Should we follow the same format for docs as our generated docs?

Reply:
changed it.